.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,18 +2,14 @@
 > Uncomment only one ` /kind` line, and delete the rest.
 > For example, `> /kind bug` would simply become: `/kind bug`
 
-> /kind api-change
 > /kind bug
-> /kind cleanup
-> /kind deprecation
-> /kind design
-> /kind failing-test
 > /kind feature
-> /kind flake
-> /kind code-refactoring
+> /kind cleanup
+> /kind tests
 >
 > Documentation changes: CI will automatically skip test runs if it is a doc related change.
 > /kind documentation
```

Comment:
Please put `/kind documentation` along with the other labels. I'm asking because putting it separate made sense when we had dependency on `skip-ci`. Having the note for doc changes after the labels should be just fine, IMO.

Reply:
Ack, updating